src/js/pre-logic.js

```diff
@@ -113,13 +182,13 @@ function crunchSprite(parentState) {
     if (isAnEdge) {
       switch(direction) {
       case 'left':
-        return {...state, x: parentState.board.rows};
+        return {...state, x: parentState.board.rows - 1};
```

Comment:
But you are only using `parentState` for rows? If you want to hoist the `rows` then it's fine. if you're using it to keep track of the previous state, it's not working.